tests/logging/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(EkatCreateUnitTest)
+
+# Test units framework
+EkatCreateUnitTest(logger logger_tests.cpp LIBS ekat)
+
+EkatCreateUnitTest(mpilogger mpi_logger_tests.cpp
+  LIBS ekat
+  MPI_RANKS 1 ${EKAT_TEST_MAX_RANKS}
```

Comment:
Do you think it's worth adding a 2-rank case to make sure logging works with > 1 process?

Reply:
I believe EKAT_TEST_MAX_RANKS defaults to 4

Reply:
Ah, my eyes sailed right past that. Thanks.